requirements.txt

```diff
@@ -6,4 +6,4 @@ dill>=0.3
 python-dateutil>=2.8.0
 stevedore>=3.0.0
 typing-extensions
-symengine>=0.11
+symengine>=0.11
```

Comment:
For those staring at this for too long to figure out what the diff is (I pulled it locally and compared the files manually), this PR is removing a trailing line feed from this line. I guess github doesn't render this well.